backend/src/src-moderation/src/main/java/org/eclipse/sw360/moderation/db/ModerationDatabaseHandler.java

```diff
@@ -239,6 +244,30 @@ public RequestStatus createRequest(Release release, User user, Boolean isDeleteR
         return moderators;
     }
 
+    private ModerationRequest appendCommentRequestingUserToRequest(ModerationRequest request, String comment) {
+        /* For the case that the user is enlarging an existing moderation request, do not overwrite the old comment,
+           but rather append the new comment to the old one. */
+        if(request.isSetCommentRequestingUser()) {
+            String totalComment = request.getCommentRequestingUser() + System.lineSeparator() + comment;
+            request.setCommentRequestingUser(totalComment);
+        } else {
+            request.setCommentRequestingUser(comment);
+        }
+        return request;
+    }
+
+    private ModerationRequest fillRequestWithCommentAndGroupOfUser(ModerationRequest request, User user) {
+        if(user.isSetCommentMadeDuringModerationRequest()) {
+            request = appendCommentRequestingUserToRequest(request, user.getCommentMadeDuringModerationRequest());
+        } else {
+            request = appendCommentRequestingUserToRequest(request, "");
+        }
+        if(user.isSetUserGroup()){
+            request.setRequestingUserGroup(user.getUserGroup());
```

Comment:
Wait a second! Aren't you supposed to save user's department (now confusingly called Group in the UI) in the moderation request? `UserGroup` is actually his role (USER, ADMIN, etc). Why would you want to store _that_ in a moderation request?

I'm afraid you've been confused by the issue text, which talks about user group. What is meant is surely the department. Sorry about the confusion, but I think you have some more work to do on this PR.

Reply:
Ah ok, that might be possible. But changing this again would also imply changing the migration script.  So, I suggest to wait for @mcjaeger to clarify what he indented?  (since he already approved the functionality...)